gradle-plugin/kubernetes/src/main/java/org/eclipse/jkube/gradle/plugin/task/AbstractJKubeTask.java

```diff
@@ -97,18 +97,26 @@ public final void runTask() {
     run();
   }
 
+  @Internal
+  @Override
+  public KubernetesExtension getExtension() {
+    return kubernetesExtension;
+  }
+
   @Internal
   protected String getLogPrefix() {
     return DEFAULT_LOG_PREFIX;
   }
 
-  public List<ImageConfiguration> customizeConfig(List<ImageConfiguration> configs) {
+  private List<ImageConfiguration> customizeConfig(List<ImageConfiguration> configs) {
     kitLogger.info("Running in %s mode", kubernetesExtension.getRuntimeMode().getLabel());
-    if (kubernetesExtension.getRuntimeMode() != RuntimeMode.OPENSHIFT) {
-      kitLogger.info("Building Docker image in Kubernetes mode");
+    if (kubernetesExtension.getRuntimeMode() == RuntimeMode.OPENSHIFT) {
+      kitLogger.info("Using [[B]]OpenShift[[B]] build with strategy [[B]]%s[[B]]",
```

Comment:
Why have you added `[[B]]..[[B]]` placeholders again here? Do we plan to support these ?

Reply:
This is what we have on the Mojo(s).

In scope of #867, yes we should. If no support is possible, then we need to figure out a better fix, since there are ANSI escape codes scattered all around (jkube-kit).